exercises/practice/acronym/test-words.8th

```diff
@@ -52,3 +66,35 @@ needs console/loaded
    then 
 ;
 
+\ compare arrays by testing elements with string equality
+: test_eqa \ s x w -- | s w x -- 
+   -test-count
+   isword? !if swap then
+   w:exec
+   ' s:= a:= nip nip if
+    con:green con:onBlack . space " ... OK" . con:white con:onBlack cr
+   else
+    con:red con:onBlack . space " ... FAIL" . con:white con:onBlack cr
+   then 
+;
+
+: test_map_neq \ m m -- 
+   -test-count
+   isword? !if swap then
+   a:new swap 0 swap a:! 0 a:@ nip w:exec
```

Comment:
```suggestion
   w:exec
```
I don't understand the purpose of `a:new swap 0 swap a:! 0 a:@ nip` -- it creates an array, pushes the TOS word into the array, extracts the word from the array and then removes the array. This sequence is in most of the test words except test_eqa where I removed it.

Reply:
I don't recall why I was doing it so if you've established that it's irrelevant then I don't have an issue with ripping it out.

Reply:
And having ripped it out, nothing's broken. Isn't archaeology fun.